cmd/agent/app/integrations.go

```diff
@@ -330,20 +347,38 @@ func pip(args []string) error {
 }
 
 func install(cmd *cobra.Command, args []string) error {
-	if err := validateArgs(args); err != nil {
+	if err := validateArgs(args, localWheel); err != nil {
 		return err
 	}
 
-	// Additional verification for installation
-	if len(strings.Split(args[0], "==")) != 2 {
-		return fmt.Errorf("you must specify a version to install with <package>==<version>")
-	}
-
 	cachePath, err := getTUFPipCachePath()
 	if err != nil {
 		return err
 	}
 
+	pipArgs := []string{
+		"install",
+		"--cache-dir", cachePath,
+		// We replace the PyPI index with our own by default, in order to prevent
+		// accidental installation of Datadog or even third-party packages from
+		// PyPI.
+		"--index-url", tufIndex,
+		// Do *not* install dependencies by default. This is partly to prevent
+		// accidental installation / updates of third-party dependencies from PyPI.
+		"--no-deps",
+	}
+
+	if localWheel {
+		// Specific case when installing from locally available wheel
+		// No compatibility verifications are performed, just install the wheel (with --no-deps still)
+		return pip(append(pipArgs, args[0]), true)
```

Comment:
There are a couple of things which I believe are by design: like how we do not perform the `pipCheck` for local wheels, and there's therefore no attempt to rollback if a wheel doesn't match the expected wheel base, etc. Are these enough differences that a message should be posted to stdout/err? This will also open the door to customers installing third-party libs, as well as checks,... right? Is this of any concern? I guess we're assuming the customer owns the code so its fine, but just wanted to make sure this isn't oversight.

Reply:
The command should still prevent the _automatic_ download and installation of third-party deps through the `pip` cmd, but yes, customers who use this feature are on their own, and perhaps we should adjust the release notes to reflect that, or warn them when they use the command.

Reply:
Also, as we discussed offline, we will perform minimal validation that the user is trying to install a agent check, and use a constraint file to prevent them from modifying the python env.

Reply:
While the `final_constraints.txt` is a good idea, note that it does not prevent users from adding _extra_ third-party dependencies that did not come baked with the agent. I think everyone here is aware of this, but I just wanted to put it out there.

Reply:
Yeah this is what we try to minimize by inspecting the wheel and making sure that `datadog_checks_base` is a dependency. If it is, then the chances are this is an agent check wheel, and not some random package.